### DISCUSSION TITLE
TOKYO, an exciting remix of technology, dance and Japanese vibes

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#15180] [Show and tell]
![IMG_3615](https://github.com/OrchardCMS/OrchardCore/assets/61869142/8e6dd909-5b88-4469-bf03-758c085793d3)

I’m glad to share my case study about [TOKYO website](https://www.tokyodancemusic.com/en/home), developed by me with the extensive use of OrchardCore. It has just been published on [ShowOrchard gallery](https://showorchard.com/gallery/music/arts/tokyo-dance-music).

[TOKYO](https://www.tokyodancemusic.com/en/home) is an Italian artist who produces dance music for bringing you joy and escape from a negativity and aggressive everyday life (it’s profoundly influenced by his journey experience in Japan). Notably, [TOKYO](https://www.tokyodancemusic.com/en/home) has gained recognition for unofficial remixes with thousands of views on YouTube ([Depeche Mode](https://www.youtube.com/watch?v=UWFhOIKHc7Q), [Ava Max](https://www.youtube.com/watch?v=e7lBKOqWaHU), [Miles Away](https://www.youtube.com/watch?v=QYGxyq2G-Ig)).

The article demonstrates the potential of OrchardCore and represents my thanks to the community for its dedication and collaborative efforts. I also would like to remember the late Jean-Thierry Kéchichian (@jtkech), who gave me support for some of my issues.

If you are curious about the [TOKYO website](https://www.tokyodancemusic.com/en/home), or you want know the potential of OrchardCore, read the article clicking on this [link](https://showorchard.com/gallery/music/arts/tokyo-dance-music).

Your comments are appreciated.

Thanks @Piedone for sharing my case study.

**Comment by duxonline** [comment]
Good job! Thank you for sharing.

**Reply by DotCat1985** [reply]
Thank you

**Comment by aaronamm** [comment]
Thanks for sharing. Nice work. 🎉🎉🎉

**Reply by DotCat1985** [reply]
Thank you 😊

**Comment by hishamco** [comment]
@DotCat1985 are there lessons to learn and what are the modules that you worked on? You can share the experience you had with others

**Reply by DotCat1985** [reply]
You can read the [article about TOKYO website](https://showorchard.com/gallery/music/arts/tokyo-dance-music), where I mentioned the modules used for that project. I installed and configured them basing on [Orchard Core Documentation](https://docs.orchardcore.net/en/latest/), [Orchard Skills videos](https://www.youtube.com/@OrchardSkills/videos) and [OrchardCore discussions](https://github.com/OrchardCMS/OrchardCore/discussions).

**Reply by hishamco** [reply]
How did are you using the Health Checks module in this site?

**Comment by hishamco** [comment]
@DotCat1985 how are the website links aware of the content localization in your website?

**Reply by DotCat1985** [reply]
See the links below from IKEA: 

SWITZERLAND - FRENCH
https://www.ikea.com/ch/fr/p/malm-cadre-lit-haut-4rgt-blanc-s99931611

AUSTRIA - GERMAN
https://www.ikea.com/at/de/p/malm-bettgestell-hoch-weiss-s29931596

 Both links redirect to the same content, but they differ from country and language.

**Reply by hishamco** [reply]
I know that :) what I mean in the main menu is how the links changed when you switch the culture

How 'fr/About` will change to `de/About` while the current visited page is not `About` page?

**Reply by DotCat1985** [reply]
There isn't about page, but I report customer service page URLs.

SWITZERLAND - FRENCH
https://www.ikea.com/ch/fr/customer-service/services/delivery/

AUSTRIA - GERMAN
https://www.ikea.com/at/de/customer-service/services/delivery/

**Reply by hishamco** [reply]
I mentioned About as an example, in your case "Customer-Service", how the link changes to the proper page when the culture has been changed

**Reply by DotCat1985** [reply]
> I mentioned About as an example, in your case "Customer-Service", how the link changes to the proper page when the culture has been changed

And so?

**Reply by hishamco** [reply]
How the links in the menu aware when you change the current culture?
